Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,21 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return tasks
+    .map(task => task.duration / 60)
+    .filter(hour => hour >= 2)
```

Comment:
`hour` is a bit to terse here. I would call it `durationInHours`.

Reply:
Thanks for the warning, I should have been more careful. I implemented the change as suggested.